README.md

```diff
@@ -1 +1,62 @@
-# microverse-week-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Set up the project, the goal here is to master all of the tools and best practices about Linter and Gitflow.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@hendridg](https://github.com/hendridg)
+- Twitter: [@hendridg](https://twitter.com/hendridg)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hendridg)
```

Comment:
- [ ] I can see you are still using the link of the README TEMPLATE. I think you forgot to put your own information. Kindly go ahead and put your valid links